motr/st/utils/motr_rmw_st.sh

```diff
@@ -120,9 +120,10 @@ write_and_update()
 {
 	local LID=$1
 	local update_count=$2
+	local di_flag=$4
 
 	echo "m0cp"
-	$motr_st_util_dir/m0cp -G $MOTR_PARAMS -o $object_id $src_file \
+	$motr_st_util_dir/m0cp $di_flag $MOTR_PARAMS -o $object_id $src_file \
```

Comment:
This Codacy warning can be skipped/ignored, I think.

Reply:
done/Ignored